File: app/src/main/java/com/example/water/cproject/DBHelper/CodeDBHelper.java
package com.example.water.cproject.DBHelper;

import android.content.Context;

/**
 * Created by watering on 18. 3. 19.
 */

@SuppressWarnings("DefaultFileTemplate")
public class CodeDBHelper extends DBHelper {
    public CodeDBHelper(Context context) {
        super(context);

        TABLE_NAME = "tbl_code";
        COLUMNS = new String[] {"code TEXT","date TEXT"};
    }
}

File: app/src/main/java/com/example/water/cproject/dbHelper/CodeDBHelper.kt
package com.example.water.cproject.dbHelper

import android.content.Context

/**
 * Created by watering on 18. 3. 19.
 */

class CodeDBHelper(context: Context) : DBHelper(context) {
    init {
        tableName = "tbl_code"
        columns = arrayOf("code TEXT UNIQUE", "date TEXT")
    }
}
